cpp/main.cc

```diff
@@ -60,12 +60,12 @@ int main(int argc, char **argv) {
   int HEMCheck            = ( argc > 21 ? char2int(argv[21]) : 0 ); // Set to 1 to check
   // Option to reproduce the summary.json
   int onlyCreateJSON      = ( argc > 22 ? char2int(argv[22]) : 0 );
+  int enrichDY            = ( argc > 23 ? char2int(argv[23]) : 0 ); // By default, don't enrich the DY
 
   // Sets variables to run the preselection on the DY enriched input datafiles 
-  bool enrich_DY = false;
   if (sampleArg == "enrichDY") {
```

Comment:
I was expecting to run the inverted-vetoes configuration with
`./main.exe <output_dir> Data ... 1`.
Now both the above and
`./main.exe <output_dir> enrichDY ... 0`
work, which is redundant and maybe a bit counterintuitive. Is this on purpose?

Reply:
I was trying to make it so that the crab submission would work with the first one, but be able to use the second one for local submissions. I can see how that might be confusing to the user, but I was trying to avoid having to type out the 15/16 int flags, which I thought might be more error prone